updater/schema.py

```diff
@@ -13,7 +13,13 @@ def checkSchemaVersion(dataDirectory):
 	schemaVersionLocal = 0
 
 	try:
-		with open(os.path.join(dataDirectory, "meta.tsv"), "r") as tsvFile:
+		metaFilePath = os.path.join(dataDirectory, "meta.tsv")
+
+		# If the data repository is empty, no schema compatibility check is necessary
+		if not os.path.exists(metaFilePath):
```

Comment:
I’d rather adjust the code, because it wouldn’t make sense to check the data repository in this case. I agree that the comment could have been more concise. I’ll adjust this.